### PR TITLE
docs: Create CONTRIBUTING.md file

### LINKED ISSUE #2: Create CONRIBUTING.md file
**Is your feature request related to a problem?**
`CONTRIBUTING.md` is the file where, by convention, new contributors can learn how to help develop the project. 
It should include information like:
– it is necessary to follow the Issue and Pull Request templates
– use only the latest `stable` version of Flutter

There's definitely more to add there but those are the few I can think of currently. 

**Describe the solution you'd like**
create `CONTRIBUTING.md` file

**Describe alternatives you've considered**
none

### PR DESCRIPTION
**Pull request by gaurivn**
### Description
Created CONTRIBUTING.md file.

Fixes #2 

### Flutter Channel:
- [x] I have used the Flutter Beta channel on my local machine 

### Type of Change:
**Delete irrelevant options.**

- Documentation

**Code/Quality Assurance Only**

### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+* Remember that this is an inclusive community, committed to creating a safe, positive environment.  See the full [Code of Conduct](http://www.systers.io/code-of-conduct.html).
+* Follow our [Commit Message Style Guide](https://github.com/anitab-org/mentorship-android/wiki/Commit-Message-Style-Guide) when you commit your changes.
+* Please consider raising an issue before submitting a pull request (PR) to solve a problem that is not present in our [issue tracker](https://github.com/anitab-org/mentorship-flutter/issues). This allows maintainers to first validate the issue you are trying to solve and also reference the PR to a specific issue.
+* When developing a new feature, include at least one test when applicable.
+* When submitting a PR, please follow [this template](.github/PULL_REQUEST_TEMPLATE.md) (which will probably be already filled up once you create the PR).
+* Use only the latest stable version of Flutter.
```
Comment (techno-disaster) [inline CONTRIBUTING.md]: beta version

**Review by techno-disaster — CHANGES_REQUESTED** [review]

**Comment by techno-disaster** [comment]
@isabelcosta please review this, and confirm if any changes are required

File: CONTRIBUTING.md (hunk 1 above)
Comment (gaurivn) [reply CONTRIBUTING.md]: @Techno-Disaster  done.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Contributing Guidelines
+
+* You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/).  Each active repo has its own stream to direct questions to (for example #powerup or #portal).  Mentorship System stream topic - Flutter is [#mentorship-flutter](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system/topic/Flutter).
```
Comment (isabelcosta) [inline CONTRIBUTING.md]:
```suggestion
* You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/).  Each active repo has its own stream to direct questions to (for example #powerup or #portal).  Mentorship System stream is [#mentorship-system](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system).
```

Even though there is a flutter topic, to discuss the project in itself as a product as a whole and its counterparts, you just need to know about the stream. The topics can change over time and all.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+* When submitting a PR, please follow [this template](.github/PULL_REQUEST_TEMPLATE.md) (which will probably be already filled up once you create the PR).
+* Use only the latest beta version of Flutter.
+* When submitting a PR with changes to user interface (e.g.: new screen, ...), please add screenshots to the PR description.
+* When you are finished with your work, please squash your commits otherwise we will squash them on your PR (this can help us maintain a clear commit history). 
+* When creating an issue to report a bug in the project, please follow our [bug_report.md](.github/ISSUE_TEMPLATE/bug_report.md) template.
+* Issues labeled as “First Timers Only” are meant for contributors who have not contributed to the project yet. Please choose other issues to contribute to, if you have already contributed to these type of issues.
+
+## General Guidelines
+
+* If you’re just getting started work on an issue labeled “First Timers Only” in any project. Additional resources are available on our [website](http://www.systers.io).
```
Comment (isabelcosta) [inline CONTRIBUTING.md]:
```suggestion
* If you’re just getting started work on an issue labeled “First Timers Only” in any project.
```
Systers.io is going to be sunset, so no need to add that :)

**Review by isabelcosta — CHANGES_REQUESTED** [review]
Two minor changes and this is fine :D 
Thank you for your contribution @gaurivn !

**Comment by gaurivn** [comment]
@isabelcosta ,thank you, I made changes, please review them, you are welcome.

**Comment by techno-disaster** [comment]
@gaurivn once you update the requested changes please re request a review so that we can review it and merge it asap

**Comment by gaurivn** [comment]
@Techno-Disaster , I have made the requested changes. I will request for review.

**Review by techno-disaster — APPROVED** [review]

**Review by isabelcosta — APPROVED** [review]
Great work @gaurivn 🎉

**Comment by gaurivn** [comment]
Thank you @isabelcosta

**Pull request merged**